### PR TITLE
[WIP] Add tsm to archivematica for Debian 9 onwards

### PR DESCRIPTION
**Pull request by sulemandiwan**

### CODE REVIEW

File: manifests/role/archivematica.pp [hunk 1]
```diff
@@ -13,6 +13,12 @@
   include nebula::profile::duo
   include nebula::profile::networking
 
+  # TSM to be included via puppet starting with Debian 9 onwards
+  if $facts['os']['family'] == 'Debian' \
```
Comment (skorner) [inline manifests/role/archivematica.pp]: Archivematica systems are all Ubuntu. Also, there are no older puppet-managed instances, so all with this role can have the TSM profile.

**Comment by skorner** [comment]
The TSM aspect is minor and looks good to me. However, I can't speak to the testing portion. I'd like @botimer or @daaang to at least glance at that and sign off.

File: manifests/profile/tsm.pp [hunk 2]
```diff
@@ -41,7 +41,8 @@
   Array[String] $virtualmountpoints = ['/etc','/opt','/var'],
   Array[String] $exclude_dirs = ['/afs/','/net/','/nfs/','/usr/vice/cache/']
 ) {
-  package { 'tivsm-ba': }
+
+  ensure_packages(['tivsm-ba', 'tivsm-api64', 'gskcrypt64', 'gskssl64',])
```
Comment (skorner) [inline manifests/profile/tsm.pp]: We should explore why requiring the single package was not enough to pull in the other three as dependencies. It worked on Debian Stretch.

Comment (sulemandiwan) [reply manifests/profile/tsm.pp]: Going to double-check this in case it was an artifact of other transient issues.

**Pull request merged**